js/wee.history.js

```diff
@@ -367,6 +389,15 @@
 			request.url = url;
 			conf.request = request;
 
+			if (conf.begin && W.$exec(conf.begin, {
```

Comment:
This is mainly about having the completed, fully built-out config object for evaluation in conjunction with a begin hook or with the go interface and a pending request. Could be the replaceState could be rearranged without mitigating any of the advantages.